Sage/carbon
# chore(select): ensure test stories callbacks work and cleanup controls

### Proposed behaviour



Remove "actions" in SimpleSelect test stories. 
Cleanup controls in all Select test stories. 

### Current behaviour



Callback events don't work in SimpleSelect test stories as they are overridden by "actions" addon. 

### Checklist




#### QA


### Additional context



### Testing instructions



See [SimpleSelect test story](https://carbon.sbc-ui.com/?path=/story/select-test--simple-select-component) onChange not working.
